### ISSUE TITLE
about og:image

### ISSUE THREAD

**Issue by mirik999** [nygardk/react-share#125]
Hi people,  what is wrong in my code? image and title doesnt shows
` <div>
	<Helmet>
	<meta property="og:image" content={article.image} />
        <meta property="og:title" content={article.title} />
	</Helmet>
        <FacebookShareButton url={`itechs.info${url}`}>
	<div className="share-icon">
        <img src={require('../../../lib/img/fb-icon.png')} alt="facebook-share-icon" />
	</div>
	</FacebookShareButton>
       </div>`

**Comment by aautio** [comment]
I assume your problem is not related to `react-share`. If you manually share your page, the image and title are also missing? This might happen if you have not declared `og` namespace.

Use `<html prefix="og: http://ogp.me/ns#">` as your root element and the image + content should become visible in the share dialog.

**Comment by jameskennethrobinson** [comment]
Still a bit confused on sharing an image. A working snippet of html would be great!

**Comment by mirik999** [comment]
root element you mean component ? or index.html ? in index.html i tired but no result.

**Comment by aautio** [comment]
When the Facebook bot crawls your site it should see the definitions. Depending on your setup it might be your index.html.

Have you tried https://developers.facebook.com/tools/debug/ to debug your page? Can you share a link to it?

**Comment by aautio** [comment]
Closing after 2 weeks of inactivity. @mirik999 please reopen if necessary.

**Comment by asiajacko** [comment]
Similar issue.

**Comment by asiajacko** [comment]
Similar issue, in React Helmet: 

`meta={[
                            property: 'og:image',
                            content: data.myImage.file.url,
                        },
                        {
                            property: 'og:image:secure_url',
                            content: data.myImage.file.url,
                        },...`
 etc

And there are 2 things.. When I copy my website and paste url on fb, I will get full image, link, description(so meta is working) I used Sharing Debugger and everything seems fine. But when I will click on "share" button on page, img is not showing. @nygardk is it connected to:

` function facebookLink(url, { quote, hashtag }) {
  assert(url, 'facebook.url');
  return 'https://www.facebook.com/sharer/sharer.php' + objectToGetParams({
    u: url,
    quote,
    hashtag,
  });
}`
????
The function is not returning an image. I checked really everything and no clue how to solve it. 

Thanks in advanced
![screen shot 2018-05-08 at 14 39 37](https://user-images.githubusercontent.com/19932162/39757635-c3771faa-52cd-11e8-8af0-62d8f21658e4.png)
